src/split-button-dropdown/index.js

```diff
@@ -1,11 +1,11 @@
 import React, { Component, PropTypes } from 'react';
```

Comment:
split-button dropdown and button dropdown look like they share a lot of code? future refactor maybe can abstract this duplication and in favor of composition